cg/services/order_validation_service/workflows/fastq/models/sample.py

```diff
@@ -0,0 +1,14 @@
+from cg.models.orders.sample_base import PriorityEnum, SexEnum
+from cg.services.order_validation_service.constants import ElutionBuffer
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class FastqSample(Sample):
+    concentration_ng_ul: float | None = None
```

Comment:
as in Microsalt @islean?
```suggestion
    sample_concentration: float | None = None
```

Reply:
Nope, actually two different fields (although I agree that this is a great source for confusion). The concentration_ng_ul can be seen here:

<img width="266" alt="image" src="https://github.com/user-attachments/assets/f61ba1c5-2d6e-4a2f-90a9-8fa329f7cd26">

while the sample_concentration can be seen here:

<img width="179" alt="image" src="https://github.com/user-attachments/assets/1ae4a538-c394-4946-a6b7-cd04395fd699">

Come to think of it - I think the sample_concentration has been removed from Microsalt due to the recent request by Fay?

Reply:
yes it has been removed

Reply:
Hmm, I'll remove the field from microsalt in a PR then

Reply:
unify names in another PR, leave as concentration_ng_ul

Reply:
I can remove it from Microsalt